integration-test/helpers/pages/loginPage.js

```diff
@@ -282,9 +302,16 @@ export function loginPage (page, opts = {}) {
             expect(count).toBe(0)
         }
 
+        /**
+         * @param {import('../../../src/deviceApiCalls/__generated__/validators-ts').SendJSPixelParams[]} pixels
+         */
+        async assertPixelsFired (pixels) {
+            await genericPage(page).assertPixelsFired(pixels)
+        }
+
         async assertNoPixelFired () {
             const mockCalls = await mockedCalls(page, {names: ['sendJSPixel'], minCount: 0})
-            expect(mockCalls).toHaveLength(0)
+            await expect(mockCalls).toHaveLength(0)
```

Comment:
I think I had already commented on this, the await shouldn't be needed, because mockCalls is already awaited above.